aten/src/ATen/native/native_functions.yaml

```diff
@@ -3259,16 +3259,24 @@
   python_module: nn
 
 - func: adaptive_avg_pool2d_out(Tensor output, Tensor self, IntList[2] output_size) -> Tensor
-  python_module: nn
+  dispatch:
+    CPU: AdaptiveAveragePooling2d_forward_out_cpu
```

Comment:
you don't need to give these forward names; forward is for binding to named outputs in derivatives.yaml, but these already bind to the non-forward versions.